lightningd/dual_open_control.c

```diff
@@ -2703,6 +2699,12 @@ json_openchannel_signed(struct command *cmd,
 	wallet_inflight_save(cmd->ld->wallet, inflight);
 	watch_opening_inflight(cmd->ld, inflight);
 
+	/* Only after we've updated/saved our psbt do we check
+	 * for peer connected */
+	if (!channel->owner)
+		return command_fail(cmd, FUNDING_PEER_NOT_CONNECTED,
+				    "Peer not connected");
+
```

Comment:
!!WARNING!! This will be messy after rebase, since now in master we use param_check!  So if we are simply running the `check` RPC, we still want to do this check.

I'm not sure how to handle this in general.  I mean, the command still does something, though it "fails": that just seems... weird.